packages/account-v2/src/hooks/usePOAInfo.ts

```diff
@@ -0,0 +1,53 @@
+import { useMemo } from 'react';
+import { useAuthentication, useMT5AccountsList } from '@deriv/api';
+
+const usePOAInfo = (landingCompanyShortCode: string) => {
+    const { data: authenticationData, ...rest } = useAuthentication();
+    const { data: mt5LoginList } = useMT5AccountsList();
+
+    const modifiedPOIInfo = useMemo(() => {
+        if (!authenticationData) return {};
+
+        const {
+            document,
+            is_age_verified,
+            is_allow_document_upload,
+            is_poa_address_mismatch,
+            is_poa_resubmission_allowed,
+            is_poi_needed,
+        } = authenticationData;
+
+        const isMxMlt = landingCompanyShortCode === 'iom' || landingCompanyShortCode === 'malta';
+
+        const hasRestrictedMT5Account = !!mt5LoginList?.filter(mt5_account =>
+            mt5_account?.status?.includes('poa_failed')
+        ).length;
+
+        const isPOAResubmission =
```

Comment:
There are some edge cased when it comes to IDV. Please refer to `isPOARequiredForMT5` from `packages/shared/src/utils/cfd/cfd.ts`

Reply:
This flag is used in the Verification flow to check if we need POA in the flow, not applicable here.